packages/components/src/components/PipelineRun/PipelineRun.js

```diff
@@ -162,7 +174,7 @@ export /* istanbul ignore next */ class PipelineRunContainer extends Component {
           reason,
           steps,
           succeeded,
-          taskName,
+          taskName: "taskName",
```

Comment:
I don't know what to put here, as we don't have a task name anymore.

Reply:
I've confirmed this isn't used anywhere so you can remove `taskName` from this object :+1:

Reply:
removed